Source/WebKit/NetworkProcess/Classifier/ResourceLoadStatisticsStore.cpp

```diff
@@ -1437,8 +1448,9 @@ Vector<ITPThirdPartyData> ResourceLoadStatisticsStore::aggregatedThirdPartyData(
 
     Vector<ITPThirdPartyData> thirdPartyDataList;
     const auto prevalentDomainsBindParameter = thirdPartyCookieBlockingMode() == ThirdPartyCookieBlockingMode::All ? "%"_s : "1"_s;
-    auto sortedStatistics = m_database.prepareStatement(joinSubStatisticsForSorting());
-    if (!sortedStatistics
+    auto preparedStatement = m_database.prepareStatement(joinSubStatisticsForSorting());
+    CheckedPtr sortedStatistics = &(preparedStatement.value());
```

Comment:
I don't think this is quite right. We need to check `!preparedStatement` before accessing `preparedStatement.value()`. Otherwise, we'd hit a release assert within.